src/language-js/parse/oxc.js

```diff
@@ -31,12 +34,25 @@ function createParseError(error, { text }) {
   });
 }
 
+let oxcParser;
+/**
+@param {string} filepath
+@param {string} text
+@param {ParserOptions} options
+@returns {Promise<ParseResult>}
+*/
 async function parseWithOptions(filepath, text, options) {
-  const result = await oxcParse(filepath, text, {
+  if (!oxcParser) {
+    os.availableParallelism ??= () => os.cpus().length;
+    oxcParser = await import("oxc-parser");
+  }
+
+  const result = await oxcParser.parseAsync(filepath, text, {
     preserveParens: true,
-    experimentalRawTransfer: rawTransferSupported(),
     showSemanticErrors: false,
     ...options,
+    experimentalRawTransfer:
+      options.experimentalRawTransfer ?? oxcParser.rawTransferSupported(),
```

Comment:
Unless I'm mistaken, checking `options.experimentalRawTransfer` isn't needed any more.

```suggestion
    experimentalRawTransfer: oxcParser.rawTransferSupported(),
```

Reply:
Thanks!